src/context.cpp

```diff
@@ -616,12 +639,44 @@ namespace librealsense
             if( option_id == RS2_OPTION_COUNT )
                 throw librealsense::invalid_value_exception( to_string() << "Option " << option->get_name() << " type not found" );
 
-            auto opt = std::make_shared< rs2_dds_option >( option );
+            auto opt = std::make_shared< rs2_dds_option >( option,
+                [&]( const std::string & name, float value ) { set_option( name, value ); },
+                [&]( const std::string & name ) -> float { return query_option( name ); } );
             register_option( option_id, opt );
         }
 
-        //float get_option( rs2_option option ) const override;
-        //void set_option( rs2_option option, float value ) const override;
+        void set_option( const std::string & name, float value ) const
+        {
+            // Sensor is setting the option for all supporting streams (with same value)
```

Comment:
What does this means?
If we have depth.exposure, and color.exposure.
Same `RS2_OPTION_EXPOSURE`
We used to do 
```
color_sensor.set_option(RS2_OPTION_EXPOSURE, 1)
depth_sensor.set_option(RS2_OPTION_EXPOSURE, 2)
```

Now it will send it to all?
Maybe I am missing something but options should be binds to device / stream
We can register same option to several and control each one seperatly

Reply:
OK